msteams-platform/task-modules-and-cards/task-modules/task-modules-tabs.md

```diff
@@ -44,20 +53,36 @@ microsoftTeams.tasks.submitTask(
 
 ---
 
-You can see how invoking a task module from a tab and submitting the result of a task module works.
+The following sections explain the process of invoking a dialog from a tab and submitting the result.
+
+## Invoke a dialog from a tab
+
+> [!NOTE]
+> Starting with TeamsJS v.2.8.x, Adaptive Card-based dialogs are supported by the [dialog](/javascript/api/@microsoft/teams-js/dialog) namespace and the `tasks` namespace is now fully deprecated. The `tasks` namespace is still supported for backward-compatibility, however best practice is to update `tasks.startTask()` calls to `dialog.url.open` or `dialog.adaptiveCard.open` for HTML- and Adaptive Card-based dialogs, respectively. For more info, see the [dialog namespace](../../tabs/how-to/using-teams-client-library.md#dialog-namespace).
```

Comment:
```suggestion
> Starting with TeamsJS v2.8.x, Adaptive Card-based dialogs are supported by the [dialog](/javascript/api/@microsoft/teams-js/dialog) namespace and the `tasks` namespace is now fully deprecated. The `tasks` namespace is still supported for backward-compatibility, however best practice is to update `tasks.startTask()` calls to `dialog.url.open` or `dialog.adaptiveCard.open` for HTML- and Adaptive Card-based dialogs, respectively. For more info, see the [dialog namespace](../../tabs/how-to/using-teams-client-library.md#dialog-namespace).
```

Reply:
The content in the line is contradicting. "the `tasks` namespace is now fully deprecated. The `tasks` namespace is still supported for backward-compatibility".

Can you please verify.

Reply:
Reworded it for clarity. How about this:

Starting with TeamsJS v2.8.x, the dialog namespace supports Adaptive Card-based dialogs, and thus fully replaces the functionality of the `tasks` namespace.